shared_model/backend/protobuf/transaction_responses/proto_tx_response.hpp

```diff
@@ -54,6 +60,23 @@ namespace shared_model {
         return proto_->error_message();
       }
 
+      /**
+       * Compare priorities of two transaction responses
+       * @param other response
+       * @return:
+       *    - -1, if this response's priority is less, than other's
+       *    -  0, if it's equal
+       *    -  1, if it's greater
+       */
+      int comparePriorities(const TransactionResponse &other) const noexcept {
```

Comment:
Use `enum class` instead

Reply:
What do you mean?

Reply:
```cpp
enum class Result { LE, EQ, GT, };

Result comparePriorities(const T&) ...
```

Reply:
OK, I implemented this method looking at old C functions, strcmp etc, where it's done in this way :) will change it

Reply:
Maybe relay operator<, >, == to transactionResponse from priority directly?